go.mod

```diff
@@ -165,6 +165,7 @@ require (
 require (
 	github.com/antihax/optional v1.0.0
 	github.com/containers/image/v5 v5.25.0
+	github.com/coreos/go-oidc v2.2.1+incompatible
```

Comment:
Is there a reason this isn't the v3 version of go-oidc? This version is from 2020 and doesn't match the 2.9 version: https://github.com/rancher/rancher/blob/21ccf17faf42d21f8e5b86c97914f034ad4a0136/go.mod#L86

Reply:
Good catch. No, it was probably because I used the `github.com/coreos/go-oidc` import instead of the `github.com/coreos/go-oidc/v3/oidc`.

Fixed with cce61e3.